index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>portfolio</title>
+</head>
+<body>
+  <nav class="header-container">
+    <a  href="#" id="logo-name"> Moha </a>
+    
+    <img id="logo-icon" src="images/Icon - Menu.svg" alt="humbuger Menu">
+  
+  </nav>
+  <article class="content-container">
+    <h1 id="greet-name">Hey There.<br>
+I'm Mohamed
+    </h1>
+    <h4 id="greet-work">I'm a Software Developer</h4>
+    <p id="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+    see and have a project you need coded,
+    don’t hesitate to contact me.</p>
+  <section class="social-illustration">
+  <div id="socials">
+    <ul>
+      <li><a href="#" target="_blank"><img src="images/github.svg" alt="github icon"></a></li>
+
+      <li><a href="#" target="_blank"><img src="images/linkedin.svg" alt="linkedin icon"></a></li>
+
+      <li><a href="#" target="_blank"><img src="images/angellist.svg" alt="angellist icon"></a></li>
+
+      <li><a href="#" target="_blank"><img src="images/twitter.svg" alt="Twitter icon"></a></li>
+
+      <li><a href="#" target="_blank"><img src="images/medium.svg" alt="medium icon"></a></li>
+    </ul>
+  </div>
+ 
+</section>
+<div id="illustration"><img src="images/Illustration.png" alt="illustration"> </div>
+  </article>
+
+</body>
+</html>
```

Comment:
- [ ] Kindly make sure this background image it placed in  good place :smile:  , to stick to the Figma design , look at the requirement as client need that you should meet as much as possible :+1: 

### Your design
![Screenshot from 2023-01-20 00-51-03](https://user-images.githubusercontent.com/80458199/213580459-60005f6f-a703-4783-b9ca-d3b89cb7e7b9.png)

### Figma design

![Screenshot from 2023-01-20 00-56-16](https://user-images.githubusercontent.com/80458199/213580646-1a82ca61-ff35-4294-a983-765eec23d690.png)